src/goMain.ts

```diff
@@ -289,6 +290,30 @@ export function activate(ctx: vscode.ExtensionContext): void {
 
 	ctx.subscriptions.push(vscode.commands.registerCommand('go.get.package', goGetPackage));
 
+	ctx.subscriptions.push(vscode.commands.registerCommand('go.lint.package', () => {
+		let editor = vscode.window.activeTextEditor;
+		let documentUri = editor ? editor.document.uri : null;
+		let goConfig = vscode.workspace.getConfiguration('go', documentUri);
+
```

Comment:
let's move all this logic inside `lintCurrentPackage` function. This is to be consistent with all the other commands and to keep the `goMain.ts` file clear
Same for `lintWorkspace` function